.circleci/config.yml

```diff
@@ -662,17 +688,20 @@ workflows:
             - build-nightly-buster-securedrop-workstation-config
       - build-nightly-buster-securedrop-workstation-grsec:
           requires:
-            - build-nightly-buster-securedrop-workstation-config
+            - build-nightly-buster-securedrop-keyring
       - build-nightly-bullseye-securedrop-log:
           requires:
-            - build-nightly-buster-securedrop-log
+            - build-nightly-buster-securedrop-workstation-grsec
+      - build-nightly-bullseye-securedrop-client:
+          requires:
+            - build-nightly-bullseye-securedrop-log
       - build-nightly-bullseye-securedrop-keyring:
           requires:
-            - build-nightly-buster-securedrop-keyring
+            - build-nightly-bullseye-securedrop-client
```

Comment:
I see we were basing the Bullseye `keyring` on the Buster `keyring`, but that doesn't seem necessary anymore since the `securedrop-client` is now available. :white_check_mark:

Reply:
Same for `workstation-grsec` :white_check_mark: